.github/workflows/ci.yml

```diff
@@ -0,0 +1,27 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: detect-secrets-ci
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+
+jobs:
+  main:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest]
+        python: ['3.6', '3.7', '3.8', '3.9']
```

Comment:
python3.6 is end-of-life, do we still need to support it?

other than that, looks good! thanks for doing this :)

Reply:
Right. Removing 3.6 was also a recommendation I got before. Will remove it from here.

Reply:
Leaving py36 in for now until we have dependencies that require 3.7+ only.